lib/ProductOpener/Config_off.pm

```diff
@@ -410,18 +410,21 @@ $options{categories_considered_as_beverages_for_nutriscore} = [qw(
 )];
 
 $options{categories_exempted_from_nutriscore} = [qw(
+	en:alcoholic-beverages
+	en:aromatic-herbs
 	en:baby-foods
 	en:baby-milks
-	en:meal-replacements
-	en:alcoholic-beverages
+	en:chewing-gum
 	en:coffees
-	en:teas
-	en:herbal-teas
 	en:food-additives
+	en:herbal-teas
 	en:honeys
-	en:vinegars
+	en:meal-replacements
 	en:salts
 	en:spices
+	en:sugar-substitutes
+	en:vinegars
+	en:teas
```

Comment:
We have too many ice teas, tea beverages that are (badly) categorized in en:teas, so we should not add it for now. Teas that don't have nutrition facts won't have nutriscore computed anyway.

Reply:
What is the right category for these?